.github/workflows/build.yaml

```diff
@@ -72,6 +72,13 @@ jobs:
           name: jvm-native-libraries
           path: zipline/src/jvmMain/resources/*
           if-no-files-found: error
+      
+      - uses: actions/upload-artifact@v4
```

Comment:
this seems to [create artifacts with the same name for different flows](https://github.com/cashapp/zipline/actions/runs/7227684902/job/19695773908?pr=1193#step:9:15), looking

Reply:
updated, figured out, added matrix suffixes, and they work well:  https://github.com/cashapp/zipline/actions/runs/7227981446
<img width="1362" alt="Screenshot 2023-12-15 at 3 44 39 PM" src="https://github.com/cashapp/zipline/assets/57155915/f2ebe822-c30f-4a88-879f-a7e6dbb9e824">